SPECS/protobuf/protobuf.spec

```diff
@@ -53,24 +57,68 @@ Provides:       %{name}-python3 = %{version}-%{release}
 %description -n python3-%{name}
 This contains protobuf python3 libraries.
 
+%package        java
+Summary:        protobuf java lib
+Group:          Development/Libraries
+BuildRequires:  chkconfig
+BuildRequires:  maven
+BuildRequires:  msopenjdk-11
+Requires:       %{name} = %{version}-%{release}
+Requires:       msopenjdk-11
+
+%description    java
+This contains protobuf java libraries.
+
 %prep
 %autosetup
+# populate m2 cache
+mkdir /root/.m2
+pushd /root/.m2
+tar -xf %{SOURCE1} --no-same-owner
+popd
 
 %build
 %configure --disable-silent-rules
+export JAVA_HOME=$(find %{_lib}/jvm -name "openjdk*")
```

Comment:
Using the `%{java_home}` macro is a safer bet. And also a bit cleaner. We're slowly trying to switch to using that instead of `find` and other approaches. You'd need an additional `BuildRequires:  javapackages-local-bootstrap`.